### ISSUE TITLE
Rendering of TeX is different with markdown preview and inside Jupyter notebook cell

### ISSUE THREAD

**Issue by coxackie** [microsoft/vscode#128490]
## Environment data

-   VS Code version: 1.58
-   Jupyter Extension version: 2021.8.0 (8 July 2021)
-   Python Extension version (available under the Extensions sidebar): N/A
-   OS: Arch Linux
-   Python and/or Anaconda version: N/A
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): N/A
-   Jupyter server running: N/A

## Expected behaviour

Rendering of TeX should be the same with markdown renderer and inside a jupyter notebook

## Actual behaviour

Rendering of TeX appears different with markdown renderer and inside a jupyter notebook

## Steps to reproduce:

Consider the following markdown text:
```
### 1.1. Starters

- Sequence of binomial models, indexed by the frequency of time periods. 
- Time unit is a year.

---

Pick *large* $N \in \mathbb{N}$, and split time into subintervals of length
$$
\Delta t = \frac{1}{N}.
$$

Our time set will be
$$
\boxed{
\mathbb{T} := \left\{ \frac{k}{N}; \, k = 0, 1, \ldots  \right\} = \{k \Delta t; \, k = 0, 1, \ldots  \}.
}
$$
- Increase in $N$ implies higher frequency of sampling times in $\mathbb{T}$.
```
If I preview it with markdown preview, I get this beatiful render:
![Screenshot from 2021-07-12 10-15-56](https://user-images.githubusercontent.com/22049642/125264343-af42ee80-e2fb-11eb-88f3-38bba7b0407d.png)

However, inside a Jupyter notebook markdown cell it renders like this:

![Screenshot from 2021-07-12 10-16-41](https://user-images.githubusercontent.com/22049642/125264397-bcf87400-e2fb-11eb-894b-64a7a7e3199a.png)

As you can see, things like `\left` and `\right` directves to TeX are not being honoured in the latter case. This has also been tested with parenthesis, etc.

**Comment by scaomath** [comment]
I encountered the same problem after updating to 1.58.

- VS Code Version: 1.58.0
- OS Version: Ubuntu 20.04
- Python extension version: v2021.6.944021595
- Jupyter extension version: v2021.8.1013163132
- All other extensions disable in workspace

Using the new native Jupyter editor (preview mode), the MathJax renders as @coxackie experienced:

![Screenshot from 2021-07-13 10-26-43](https://user-images.githubusercontent.com/4467042/125483692-292a9c0d-bc45-419f-ba96-fb7db053b184.png)

Switching back to the custom editor by doing
```json
"jupyter.experiments.optOutFrom": [
         "NativeNotebookEditor"
    ],
```
gives the correct render of `\left( \right)`:
![Screenshot from 2021-07-13 10-29-34](https://user-images.githubusercontent.com/4467042/125483924-120bf6b3-d3e6-4b00-86a3-1f257e16ae15.png)

Interestingly enough, on a "slow" computer, using the old custom Jupyter viewer, the notebook will first be rendered like the one in the new native editor, then become the better looking `KaTeX`-rendered formula as above.

**Comment by mjbvz** [comment]
Please give this a try in the first 1.60 insiders builds and let me know if there's still any issues 

The root cause was that the math fonts were not loading properly

**Comment by coxackie** [comment]
I am currently on vacation and will not be able to check this for some weeks. @scaomath could you possible check if the issue is resolved using insiders?

**Comment by scaomath** [comment]
> I am currently on vacation and will not be able to check this for some weeks. @scaomath could you possible check if the issue is resolved using insiders?

Somehow https://code.visualstudio.com/insiders/ keeps giving me 1.59...@mjbvz

**Comment by scaomath** [comment]
@coxackie 
I can confirm that this bug has been fixed in 1.60 with all extension disabled (except Python/Jupyter).

Only the author can use /verified to add the label, would you mind add the label @mjbvz ?

Can't wait to use it in the regular version (I am using Jupyter in VSCode teaching classes).
